src/Medusa.ts

```diff
@@ -45,12 +46,9 @@ export class Medusa {
 	 * @param modules The modules to load into medusa
 	 */
 	public async load(modules: Type[]): Promise<MedusaContainer> {
-		const { configModule } = getConfigFile(this.#rootDir, 'medusa-config') as {
-			configModule: {
-				monitoring: MonitoringOptions;
-			};
+		const configModule = (await asyncLoadConfig()) as unknown as {
```

Comment:
Question: would it be nice to make the utils template so that the return type is specified instead of force cast the type?

Reply:
done